tests/test_sqs/test_sqs.py

```diff
@@ -2283,3 +2283,88 @@ def test_send_message_fails_when_message_size_greater_than_max_message_size():
     ex.response["Error"]["Message"].should.contain(
         "{} bytes".format(message_size_limit)
     )
+
+
+@mock_sqs
+def test_fifo_queue_send_multiple_duplicate_messages():
```

Comment:
We should be able to merge all these tests into one or two using [pytest.mark.parametrize](https://docs.pytest.org/en/stable/parametrize.html)

Also, I see one test-case missing: Different `MessageBody`'s with same `MessageDeduplicationId` should give a queue length of `1`

Reply:
made 2 parameterized tests
kept 2 independent tests for deduplication time and sha256 deduplication id. I did this for better readability.